forte/processors/elastic_search_index_processor.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=attribute-defined-outside-init
+from typing import Dict, Any
+
+from texar.torch.hyperparams import HParams
+
+from forte.common.resources import Resources
+from forte.processors.base import IndexProcessor
+from forte.indexers import ElasticSearchIndexer
+from forte import utils
+
+__all__ = [
+    "ElasticSearchIndexProcessor"
+]
+
+
+class ElasticSearchIndexProcessor(IndexProcessor):
```

Comment:
Do we need to benchmark the speed of the indexer? Hopefully, our wrapper won't decrease the speed a lot.